webpack.config.js

```diff
@@ -20,35 +20,56 @@ const { getIfUtils, removeEmpty } = require( 'webpack-config-utils' );
 module.exports = ( env ) => {
   const { ifProd, ifNotProd, ifTest, ifNotTest } = getIfUtils( env );
 
+  function _packagePrefix() {
+    return './packages' + ( env.element ? env.element : '' );
+  }
+
+  function _testModules() {
+    return {
+      'test': _packagePrefix() + '/index.test.ts',
+      'test-helpers' : './test-helpers.ts'
+    };
+  }
+
+  function _prodModules() {
+    return {
+      'index': _packagePrefix() + '/index.ts',
+      'index-with-dependencies': _packagePrefix() + '/index.with.dependencies.ts',
+      'styles': './styles.ts'
+    };
+  }
+
+  function _devModules() {
+    return {
+      'main.demo': _packagePrefix() + '/index.demo.ts',
+      'polyfills': './polyfills.ts',
+      'styles': './styles.ts'
+    };
+  }
+
+  function _entryPoint() {
+    if ( ifTest() ) {
+
+      return _testModules();
```

Comment:
sorry I didn't realize that earlier but we should move returns from  _devModules, _prodModules and _testModules directly here so it's more straightforward. You can even put IIAFE(Immediately Invoked Arrow Function Expression 😉 ) to `entry` directly so we keep structure of config more consistent.